app/utils/auth.js

```diff
@@ -0,0 +1,40 @@
+const AUTH_SECRET = 'auth-secret';
```

Comment:
@sourcesoft If you look at `checkSecret` you'll see it loads the secret via `popSecret` from `localStorage`. `AUTH_SECRET` is just the key to access `localStorage`. The secret is a random string generated in `nonce` method. Does that answer the question?